### PR TITLE
Hotfix master - Fix Sass syntax issue with clean-import-paths rule

### LINKED ISSUE #179: Sass Syntax - Clean import paths issues
There are issues with the `clean-import-paths` rule when using the sass syntax as it's valid syntax to use without quotes. See guide http://sass-lang.com/guide

This in turn breaks the way we're detecting strings and validating them so a new way will need to be thought up.

### PR DESCRIPTION
**Pull request by bgriffith**
clean-import-paths doesn't work with Sass syntax. This fixes that issue.

Fixes #179 

DCO 1.1 Signed-off-by: Ben Griffith gt11687@gmail.com

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -149,4 +149,8 @@ helpers.loadConfigFile = function (configPath) {
   return file;
 };
 
+helpers.stripQuotes = function (str) {
```
Comment (DanPurdy) [inline lib/helpers.js]: Are there tests for this coming on the develop branch?

**Comment by DanPurdy** [comment]
Seems good, just the extra helper function tests missing..

**Comment by bgriffith** [comment]
Tests added for helper function :rocket:

**Comment by DanPurdy** [comment]
:+1:

**Pull request merged**